code/datums/wires/syndicatebomb.dm

```diff
@@ -59,11 +64,13 @@
 
 /datum/wires/syndicatebomb/on_cut(wire, mend)
 	var/obj/machinery/syndicatebomb/B = holder
+	B.add_fingerprint(usr)
 	switch(wire)
 		if(WIRE_BOOM)
 			if(!mend && B.active)
 				holder.visible_message("<span class='danger'>[icon2html(B, viewers(holder))] An alarm sounds! It's go-</span>")
 				B.explode_now = TRUE
+				usr.log_message("detonated syndicate bomb via cut boom wire", LOG_ATTACK)
```

Comment:
are you sure usr will always be a player?

Reply:
I'll cast usr to user and null check at the start of the block